dough-protection/src/main/java/io/github/bakedlibs/dough/protection/ProtectionManager.java

```diff
@@ -51,31 +51,33 @@
      *
      * @param server
      *            The Server your plugin is running on.
+     * @param plugin
+     *            The plugin that uses dough.
      */
     @SuppressWarnings("java:S1612")
-    public ProtectionManager(@Nonnull Server server) {
+    public ProtectionManager(@Nonnull Server server, @Nonnull Plugin plugin) {
```

Comment:
Not a big fan of this being named `plugin` but can't think of a better name ccurrently